src/lib/blockchains/BitShares.js

```diff
@@ -9,6 +9,8 @@ import {
     Signature
 } from "bitsharesjs";
 import * as Socket from "simple-websocket";
+import _ from "lodash";
```

Comment:
do we really need to introduce lodash now?

Reply:
I could replace the lodash usage with pure javascript like https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore